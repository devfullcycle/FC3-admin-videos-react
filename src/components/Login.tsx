import React from "react";

export default function Login() {
  return (
    <div>
      <h1>Please login to access the application</h1>
    </div>
  );
}
