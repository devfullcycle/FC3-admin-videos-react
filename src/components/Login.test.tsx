import { renderWithProviders } from "../utils/test-utils";
import Login from "./Login";

describe("Login", () => {
  it("should render correctly", () => {
    const { asFragment } = renderWithProviders(<Login />);
    expect(asFragment()).toMatchSnapshot();
  });
});
