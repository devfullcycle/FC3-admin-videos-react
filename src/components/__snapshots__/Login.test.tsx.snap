// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render correctly 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Please login to access the application
    </h1>
  </div>
</DocumentFragment>
`;
